SemVer.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Created by Anita on 07/08/2016.
+ */
+
+myModuleI = {
+    SemVer: function (major, minor, revision) {
+        if (major < 0 || minor < 0 || revision < 0) {
+            throw new Error('Error: negative number' );
+        }else {
+            this.major = major;
+            this.minor = minor;
+            this.revision = revision;
+        }
+        /* Code to see if major is valid, how implements that? And then, testing:
+
+        var regex = /^[1-9][0-9][0-9]$|^[1-9][0-9]$|^[1-9]$/;
+        if(regex.test(major)){
+            return major;
+        }else{
+            return 'Value for major incorrect';
+        }
+
+        */
+
+        this.convertToString = function () {
+            var major = this.major.toString();
```

Comment:
No need to make .toString() here. When concatenating variables with "+", the toString() method is called automatically.
Here is called twice and the code is much larger.